lib/identity/realm.js

```diff
@@ -295,6 +328,8 @@ class Realm extends CryptoConsumer {
       }
     )
     let sessionStart = await credentialedDecodeResponse(request)
+    const { federated } = sessionStart
+    delete sessionStart.federated
```

Comment:
Why do we need to delete the federated flag?

Reply:
curious about this as well

Reply:
The session data is pulling double duty now.. It's not my favorite way to do this, but was a way to get it done quickly. Later all of the items in `sessionStart` get turned into query parameters, 'federated' is not needed. This pulls it into a local var and then removes it from the object for query param usage.

This can and should be much better, but timelines...

Reply:
comment to explain what is happening and suggest better structure down the road.